WordPressAuthenticator/NUX/NUXButton.swift

```diff
@@ -1,54 +1,285 @@
 import UIKit
```

Comment:
The NUXButton file is a duplicate of `FancyButton` with the activity view add-on (in case you're wondering where this code originated from)